experiment/manage_release_versions.py

```diff
@@ -27,25 +27,31 @@
 
 import re
 import collections
-
-# Probably should just read out of a config file in the future.
-
-
-class Version(object):
-    CURRENT = "1.14" # also known as beta in some other places.
-    STABLE1 = "1.13"
-    STABLE2 = "1.12"
-    STABLE3 = "1.11"
-    DEPRECATED = "1.10"
+import yaml
+
+# Constants in the releases.yaml file
+CURRENT = "current"
```

Comment:
@imkin I'm not seeing the need of having a class for the versions instead of having a dict. I think it's overkill. Could you explain why you'd like to keep the class?

Reply:
Using class as a synctactic sugar for constants is a healthy practice. In short it is to avoid globals, provide better namespacing for variables, ability to define load and import in other modules.
It also enables one to write class methods later that could help in doing operations like "verify_ version_string" " getters/setters. 

I do not think they are an overkill as the the loc is actually lesser, python likes it and is a pattern recommended in general in place global "dicts".